src/style/base/_base.scss

```diff
@@ -48,25 +58,33 @@ h1, h2, h3, h4, h5, h6 {
 }
 
 h1 {
-    font-size: 60px;
+    font-size: 110px;
```

Comment:
If this was meant to apply to the h1 tag used for displaying conference theme title "Beyond the Mind," then this css is overwritten by the h1 tag under `home-title` in `home.scss` on [line 117](https://github.com/CSSA-UCSD/conference2023/pull/14/files#diff-5b1c6bd3f37c3f6e6b17c88116c9e28c3df3723477c4d1349c58b6bfc9c0699cR117).

I think we can write the conference title css in the file linked above, and change this font-size to `70px` since that applies to the schedule page title and speakers page heading for each day?